### PR TITLE
Add shebang to main script

### PR DESCRIPTION
**Pull request by dragetd**
For all the cool scripts, need 1 shebang.

### CODE REVIEW

**Comment by davidmehren** [comment]
Sadly, I can't accept this contribution :(

File: main.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/python3
```
Comment (davidmehren) [inline main.py]: Please use `/bin/env python3`

**Review by davidmehren — CHANGES_REQUESTED** [review]
Bad!

File: main.py (hunk 1 above)
Comment (dragetd) [reply main.py]: Okay, I will fix it.

**Review by davidmehren — APPROVED** [review]
Good! \o/

**Pull request merged**